library/spdm_requester_lib/libspdm_req_get_measurements.c

```diff
@@ -525,6 +525,10 @@ static libspdm_return_t libspdm_try_get_measurement(void *context, const uint32_
 
     status = LIBSPDM_STATUS_SUCCESS;
 
+    #if LIBSPDM_ENABLE_MSG_LOG
+    spdm_context->msg_log.offset += spdm_response_size;
+    #endif /* LIBSPDM_ENABLE_MSG_LOG */
```

Comment:
It looks weird to me that this function suddenly add the offset...

I suggest we have a consistent programing mode:
 The caller should use a function to append the data to msg_log. The data appending and size adjustment should be done in one function. Not separately.

I think we need have an internal function : libspdm_append_msg_log()

Reply:
That will work if, for example, `libspdm_try_get_measurement` doesn't modify the response before it is appended. I'll add it in the next commit.